bodhi-messages/setup.cfg

```diff
@@ -0,0 +1,55 @@
+[metadata]
+name = bodhi-messages
+version = attr: bodhi.messages.__version__
+description = JSON schema for messages sent by Bodhi
+long_description = file: README.rst
+maintainer = Fedora Infrastructure Team
+maintainer_email = infrastructure@lists.fedoraproject.org
+platforms = Fedora, GNU/Linux
+url = https://github.com/fedora-infra/bodhi
+keywords = fedora, fedora-messaging
+license = GPLv2+
+# Possible options are at https://pypi.python.org/pypi?%3Aaction=list_classifiers
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: System Administrators
+    License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Topic :: System :: Software Distribution
+
+[options]
+zip_safe = False
+include_package_data = True
+packages = find_namespace:
+namespace_packages = bodhi
+install_requires =
```

Comment:
And when you do use the above pkg_resources-style namespace package, I believe you do need to runtime-require setuptools explicitly.